doc: update the topology document for external mode (#13955)
rook/rook: Documentation/CRDs/Cluster/external-cluster/topology-for-external-mode.md

volumeBindingMode: WaitForFirstConsumer
```

> Update the rook-ceph-operator-config configmap data with the key value,

Reviewer: The line numbers of those links below will be difficult to maintain. In fact, since last week they are already incorrect. Just put a link to the operator.yaml and they can see the configmap there at the top of the file.
```suggestion
Set two values in the [rook-ceph-operator-config configmap](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml):
```

Author: okay make sense, not sure if there is dynamic linking available now or in future...

> [CSI_ENABLE_TOPOLOGY: "true"](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml#L509)

Reviewer: ```suggestion
- `CSI_ENABLE_TOPOLOGY: "true"`: Enable the feature
```

> [CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml#L514)

Reviewer: ```suggestion
- `CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"`: Set the topology domain labels that the CSI driver will analyze on the nodes during scheduling.
```

Reviewer: Why would you specify host, zone, and rack topology in the same value? Shouldn't you just set one topology type?

Author: Yes for the current example set to zone
